src/ads_impl.cc

```diff
@@ -390,10 +390,11 @@ void AdsImpl::ClassifyPage(const std::string& url, const std::string& html) {
   auto winner_over_time_category = GetWinnerOverTimeCategory();
 
   LOG(INFO) << "Site visited " << url << ", immediateWinner is "
-      << winning_category << " and winnerOverTime is "
-      << winner_over_time_category;
+            << winning_category << " and winnerOverTime is "
+            << winner_over_time_category << "previous tab url "
+            << last_shown_tab_url_;
 
-  if (last_shown_tab_url_ == url) {
+  if (url.compare(last_shown_tab_url_) == 0) {
```

Comment:
`last_shown_tab_url_ == url` is the preferred way unless implementing a sorting algorithm